realtime/README.md

```diff
@@ -0,0 +1,141 @@
+# templates-realtime
+
+## Summary
+
+This Microservice demonstrates how listeners for Cumulocity realtime notifications can be implemented. Realtime notifications enable the user to listen to updates for certain Cumulocity domain objects such as alarms, operations, measurements or managedObjects.
```

Comment:
I would also add link to cumulocity online documentation for more detailed information. https://cumulocity.com/guides/reference/real-time-notifications/ also add important info from documentation "Info: The long-polling interface is designed as a mechanism for custom applications to poll infrequent events from Cumulocity IoT. The long-polling interface is not designed as a mechanism to stream large data volumes (>100kB/sec) or frequent data (>50 events/sec) out of Cumulocity IoT. The usage of long polling is not supported for such use cases."

Reply:
good point. Done